arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -43,8 +43,15 @@ struct rmid_entry {
  */
 static LIST_HEAD(rmid_free_lru);
 
-/*
- * @rmid_limbo_count - count of currently unused but (potentially)
+/**
+ * @closid_num_dirty_rmid    The number of dirty RMID each CLOSID has.
+ * Only allocated when CONFIG_RESCTRL_RMID_DEPENDS_ON_CLOSID is defined.
+ * Indexed by CLOSID. Protected by rdtgroup_mutex.
+ */
+static int *closid_num_dirty_rmid;
```

Comment:
at upstream this is u32

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev